pullrequest.js

```diff
@@ -77,6 +105,24 @@ chrome.storage.sync.get({url: ''}, function(items) {
 
         $body.on('click', '.js-collapse-deletions', collapseDeletions);
 
+        var $pullRequestTabs = $('.js-pull-request-tab');
```

Comment:
I think your problem of it working via after the ajax load is right here... if you move this selector into the keydown callback it will lookup the tabs again each time you press the key, my guess is the tabs aren't there when you load the PR list.